exchanges/graphcache/src/helpers/help.ts

```diff
@@ -1,7 +1,7 @@
 // These are guards that are used throughout the codebase to warn or error on
 // unexpected behaviour or conditions.
 // Every warning and error comes with a number that uniquely identifies them.
-// You can read more about the messages themselves in `docs/help.md`
+// You can read more about the messages themselves in `docs/graphcache/errors.md`
```

Comment:
There should be some links in this file that’ll need to be updated to point to the website :)
It may also be helpful to leave a note on the new URL and file path in the old `help.md` location

Reply:
Good point! Changes:

* Left a comment in the old `help.md`
* Updated bit.ly links